project.clj

```diff
@@ -69,10 +69,8 @@
             [lein-auto "0.1.2"]
             [lein-less "1.7.5"]
             [lein-shell "0.5.0"]
-            [cider/cider-nrepl "0.15.0-SNAPSHOT"]
```

Comment:
Since every developer has their own cider version installed, a good practice would be to have in local profiles in `~/.lein/profiles.clj`. For example, I've got 0.15.1 version installed that causes warning messages when connecting to repl.

Reply:
Could you remove this change for now? We plan to address that here: https://github.com/status-im/open-bounty/issues/255